apps/frontend/src/auth/__tests__/Signin.test.tsx

```diff
@@ -78,7 +78,7 @@ it('renders a button to signin', async () => {
 describe('when clicking the button', () => {
   beforeEach(async () => {
     const { getByText } = await renderSignin();
-    userEvent.click(getByText(/sign.+in/i));
+    userEvent.click(getByText(/sign\sin/i));
```

Comment:
Meh, don't care about individual cases but overall try not to make matchers more specific, or we'll have to change tests on absolutely every copy change at some point (e.g. 'Sign me in')